### DISCUSSION TITLE
How do I verify text-transform worked

### DISCUSSION THREAD

**Discussion by jonmhayden** [cypress-io/cypress#22625] [Questions and Help] [answered]
Hi,

I've googled this, but haven't had any luck.

Take this CSS:
`
    h1 {
        text-transform: uppercase;
    }
`

So if I had a header that was "test", it would display this:

TEST

How do I verify the text-transform worked?  Here is what I did in Cypress:

`        cy.get('h1').contains("test") // << passes `
`        cy.get('h1').contains("TEST") // << fails `

I would expect the results would be opposite since I want to verify what is actually being displayed to the user.  As a QA tester, I would want to make sure a developer didn't accidentally put in a typo to text-transform css.  Is this possible or do I have to use some sort of image-based test tool?

Thanks for the help,
Jon

**Comment by jjhelguero** [comment, marked as answer]
You can check the element text contains `text-transform by using contains with the option `matchCase: false` to find the element and assert it has `text-transform: uppercase` css.

```
cy.contains('h1', 'text', { matchCase: false }).should(
  'have.css',
  'text-transform',
  'uppercase'
)
```

**Reply by jonmhayden** [reply]
Awesome.  Thanks for that.
